ui-framework-documentation.md

```diff
@@ -58,10 +71,10 @@ use testapi;
 testapi::set_distribution(DistributionProvider->provide());
 ```
 
-DistributionProvider is a factory that returns the required Distribution
-depending on openQA environment variables ('VERSION', 'ARCH', 'BACKEND'
+DistributionProvider is a [factory](https://en.wikipedia.org/wiki/Factory_%28object-oriented_programming%29) that 
+returns the required Distribution depending on openQA environment variables ('VERSION', 'ARCH', 'BACKEND'
 etc.). Currently, Tumbleweed is returned as the default one, following
-["Factory First"](https://www.suse.com/documentation/suse-best-practices/singlehtml/sbp-quilting-osc/sbp-quilting-osc.html#sec.factory)
+["Factory First"](https://opensource.suse.com/suse-open-source-policy#factory-first)
```

Comment:
I believe we should extend the explanation of the factory, for example pointing that for example Leap inherit from latest SLE, but sometimes openSUSE in general behaves a bit different from SLE and we must overwrite functionality. But without entering in details we could just explain in general how does it work for other products different than TW.

Reply:
changed